doc/man3/d2i_X509.pod

```diff
@@ -592,6 +592,11 @@ B<i2d_I<TYPE>_bio>() and B<i2d_I<TYPE>_fp>(),
 as well as i2d_ASN1_bio_stream(),
 return 1 for success and 0 if an error occurs.
 
+On error, these functions or other functions they call may record the
```

Comment:
nit: imo there is no need to explicitly state that the functions which are called by this function may also record error code at it is implicit from that "these functions set it".